backends/oauth2_test.go

```diff
@@ -0,0 +1,338 @@
+package backends
+
+import (
+	"encoding/json"
+	"fmt"
+	. "github.com/iegomez/mosquitto-go-auth/backends/constants"
+	log "github.com/sirupsen/logrus"
+	. "github.com/smartystreets/goconvey/convey"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+	"time"
+)
+
+type oauth2ServerUserinfoResponse struct {
```

Comment:
Why do we need to declare a new struct that's almost the same us the one in the package being tested, just lacking one of the topics group?